resources/cloudwatch-alarms.go

```diff
@@ -47,6 +49,11 @@ func ListCloudWatchAlarms(sess *session.Session) ([]Resource, error) {
 	return resources, nil
 }
 
+func GetAlarmTags(svc *cloudwatch.CloudWatch, arn *string) []*cloudwatch.Tag {
+	resp, _ := svc.ListTagsForResource(&cloudwatch.ListTagsForResourceInput{ResourceARN: arn})
```

Comment:
I don't think that we should ignore the possible error here

Reply:
Fair enough - I'll bubble the errors up.